public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -589,13 +590,23 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
     const title = panel.getDisplayTitle();
     const padding: PanelPadding = plugin.noPadding ? 'none' : 'md';
 
+    let menu;
+    if (!dashboard.meta.publicDashboardAccessToken) {
+      menu = (): React.ReactElement => (
+        <div data-testid="panel-dropdown">
+          <PanelHeaderMenuWrapper panel={panel} dashboard={dashboard} onClose={() => {}} />
```

Comment:
Can we add a remaining todo do use the menu component from grafana/ui instead (in a follow up PR)

Reply:
Yes 🙌